server/bin/gold/test-7.10.txt

```diff
@@ -11257,5 +11257,5 @@ run-1970-01-01T00:00:42-UTC: Processed 1 tarballs
 ---- pbench-satellite-local/logs
 --- pbench log file contents
 +++ SqliteDB Datasets
-fake|dhcp31-44|uperf_uperftest_2018.02.02T20.58.00|INDEXED|
+fake|dhcp31-44|uperf_uperftest_2018.02.02T20.58.00|INDEXED| INDEX_MAP = (85930) {"pbench-unittests.v6.run-data.2018-02": ["4689ac905cf35910d5d110ea6724d5f3"], "
```

Comment:
Does this change belong to this PR?  (I can't tell...and I'm just assuming that the result is correct....)